### ISSUE TITLE
.NET 8.0 support

### ISSUE THREAD

**Issue by kshyju** [Azure/azure-functions-dotnet-worker#1644]
.NET 8 support tracking items.

- [x] #1653
- [x] Project template updates 
     - [x] https://github.com/Azure/azure-functions-templates/pull/1405
     - [x] Publish Microsoft.Azure.Functions.Worker.ProjectTemplates nuget package with the new bits.
- [x] https://github.com/Azure/azure-functions-docker/issues/923
     - [x] Update readme in [Azure/azure-functions-docker](https://github.com/Azure/azure-functions-docker#mcr-docs) once the images available in MCR.
- [x] #1697
     - [x] Update Linux consumption images cache list
- [x] Portal update/Stack API
- [x] Tooling support
    - [x] https://github.com/Azure/azure-functions-tooling-feed/issues/452 
- [x] Validate deployment scenarios. 
- [x] #1874

**Comment by benjaminsampica** [comment]
Any idea when we could see some preview support?

**Comment by JayAtSherweb** [comment]
Any update at all on this?

**Comment by FahaoTang** [comment]
Any updates about this? I just upgraded all my .NET related Azure projects (incl. a common shared library between Azure Container App and Azure Function App) to .NET 8 and just learnt the function app is not working.

**Comment by aherrick** [comment]
@fahaotang az func always late to the game :(

**Comment by FahaoTang** [comment]
> @FahaoTang az func always late to the game :(

I am now thinking of migrating my Azure Function App to host in Azure Container Apps (a preview feature). I noticed there is dockerfile available now for .NET 8:
https://github.com/Azure/azure-functions-docker/blob/dev/host/4/bullseye/amd64/dotnet/dotnet8-isolated/dotnet-isolated-slim.Dockerfile

**Comment by kshyju** [comment]
We are getting closer... 

https://azure.microsoft.com/en-us/updates/public-preview-azure-functions-net-8-support-in-linux-plans

**Comment by FahaoTang** [comment]
> We are getting closer...
> 
> https://azure.microsoft.com/en-us/updates/public-preview-azure-functions-net-8-support-in-linux-plans

Hello do you know when it will be available in the consumption plan? or do you know when the dotnet 8 isolated preview image will be published in MCR? if it is going to take time, I will probably use Azure Container Apps to host my dotnet 8 function app with custom image.

**Comment by kshyju** [comment]
.NET8 support is available in Linux consumption as well 😄

#### Portal update
You can now create a .net8 isolated app via azure portal.
![net-8-linux](https://github.com/Azure/azure-functions-dotnet-worker/assets/144469/c0a70bb0-9321-49c1-a86e-5d21f92d56ed)

#### Images
Images are available in mcr, though some docs updates are still in-progress as of today, but should be completed by next week.

> mcr.microsoft.com/azure-functions/dotnet-isolated:4-dotnet-isolated8.0

You can retrieve a list of all available tags for azure-functions/dotnet-isolated at https://mcr.microsoft.com/v2/azure-functions/dotnet-isolated/tags/list


There will be a few more updates coming in in the next couple of days for docs update, tooling support etc.. I will share an update on this thread as they happen.

**Comment by FahaoTang** [comment]
> .NET8 support is available in Linux consumption as well 😄
> 
> #### Portal update
> You can now create a .net8 isolated app via azure portal. ![net-8-linux](https://user-images.githubusercontent.com/144469/265304120-c0a70bb0-9321-49c1-a86e-5d21f92d56ed.png)
> 
> #### Images
> Images are available in mcr, though some docs updates are still in-progress as of today, but should be completed by next week.
> 
> > mcr.microsoft.com/azure-functions/dotnet-isolated:4-dotnet-isolated8.0
> 
> You can retrieve a list of all available tags for azure-functions/dotnet-isolated at https://mcr.microsoft.com/v2/azure-functions/dotnet-isolated/tags/list
> 
> There will be a few more updates coming in in the next couple of days for docs update, tooling support etc.. I will share an update on this thread as they happen.

This is great news. thank you very much :)

**Comment by kshyju** [comment]
Great news! You can now use [VS 2022 preview ](https://visualstudio.microsoft.com/vs/preview/)version to create .net8 Isolated function apps. ✨

![VS-Preview-net8](https://github.com/Azure/azure-functions-dotnet-worker/assets/144469/7221b409-983f-4763-b12f-0d6e14d29560)

If you are using a preview version earlier than preview3, you may need to force an update on the azure functions tools in visual studio. To do this, open "**_Tools->Options->Projects and Solutions-> Azure Functions_**" and click on the "Check for updates" button. If the UI suggests there are updated available to download, click on "Download & Install". Once this is done, you should see the net8 option in the File->New project-> Azure functions dialog.

![Vs-AzureFunctionsToolsUpdate](https://github.com/Azure/azure-functions-dotnet-worker/assets/144469/de2bbaa1-edf7-4a8f-b8b8-60b554b16989)

**Comment by kshyju** [comment]
[Azure Functions Dotnet (Isolated) DockerHub ](https://hub.docker.com/_/microsoft-azure-functions-dotnet-isolated)page is now updated with the .net8 images.

**Comment by Peter-Juhasz** [comment]
I am trying to publish a .NET 8 RC1 app to a Linux, .NET 8 Preview, Consumption plan from Visual Studio 17.8 P2 (Windows), but get the following error:
![image](https://github.com/Azure/azure-functions-dotnet-worker/assets/9047283/2487917b-75cd-4e51-819c-7859bd6eb88f)

```
Publish has encountered an error.
Publish has encountered an error. We were unable to determine the cause of the error. Check the output log for more details. 
```

I've tried both Framework-dependent and Self-contained deployments, both of them give the same error. The referenced error file doesn't contain any additional info.

```
The attempt to publish the ZIP file through https://REDACTED.scm.azurewebsites.net/api/zipdeploy failed with HTTP status code Failed.
```

**Comment by Peter-Juhasz** [comment]
I was able to publish the app manually (running `dotnet publish -r linux-x64`, zipping the results and calling `az functionapp deployment source config-zip`).

I was able to cut through the breaking differences between Windows and Linux hosting (setting App Settings via Configuration) and completely silly error messages (the runtime was looking for configurations with clearly wrong keys).

According to Azure Portal, it seems like it is working, I can see all functions listed, Enabled with green checkmarks, the queue is being consumed, etc, but none of the functions work at all.

I can see the following exception in Application Insights:
```
Failed to start a new language worker for runtime: dotnet-isolated.

An error occurred trying to start process '/home/site/wwwroot/App.Functions' with working directory '/home/site/wwwroot'. Permission denied

System.ComponentModel.Win32Exception:
   at System.Diagnostics.Process.ForkAndExecProcess (System.Diagnostics.Process, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
   at System.Diagnostics.Process.StartCore (System.Diagnostics.Process, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
   at System.Diagnostics.Process.Start (System.Diagnostics.Process, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
   at Microsoft.Azure.WebJobs.Script.Workers.WorkerProcess.StartProcessAsync (Microsoft.Azure.WebJobs.Script, Version=4.24.0.0, Culture=neutral, PublicKeyToken=null: /src/azure-functions-host/src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs:64)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Microsoft.Azure.WebJobs.Script.Grpc.GrpcWorkerChannel+<StartWorkerProcessAsync>d__61.MoveNext (Microsoft.Azure.WebJobs.Script.Grpc, Version=4.24.0.0, Culture=neutral, PublicKeyToken=null: /src/azure-functions-host/src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs:281)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Microsoft.Azure.WebJobs.Script.Workers.Rpc.RpcFunctionInvocationDispatcher+<InitializeJobhostLanguageWorkerChannelAsync>d__50.MoveNext (Microsoft.Azure.WebJobs.Script, Version=4.24.0.0, Culture=neutral, PublicKeyToken=null: /src/azure-functions-host/src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs:152)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Microsoft.Azure.WebJobs.Script.Workers.Rpc.RpcFunctionInvocationDispatcher+<InitializeJobhostLanguageWorkerChannelAsync>d__49.MoveNext (Microsoft.Azure.WebJobs.Script, Version=4.24.0.0, Culture=neutral, PublicKeyToken=null: /src/azure-functions-host/src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs:144)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Microsoft.Azure.WebJobs.Script.Workers.Rpc.RpcFunctionInvocationDispatcher+<InitializeJobhostLanguageWorkerChannelAsync>d__48.MoveNext (Microsoft.Azure.WebJobs.Script, Version=4.24.0.0, Culture=neutral, PublicKeyToken=null: /src/azure-functions-host/src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs:135)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Microsoft.Azure.WebJobs.Script.Workers.Rpc.RpcFunctionInvocationDispatcher+<>c__DisplayClass55_0+<<StartWorkerProcesses>b__0>d.MoveNext (Microsoft.Azure.WebJobs.Script, Version=4.24.0.0, Culture=neutral, PublicKeyToken=null: /src/azure-functions-host/src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs:215)
```

It consumes the queue, but all processing fails with:
```
Final functionDispatcher state: Initializing. Initialization timed out and host is shutting down
Did not find any initialized language workers
Executed 'Functions.REDACTED' (Failed, Id=080fa978-e124-4a39-9445-6524e56306e2, Duration=41051ms)
Did not find any initialized language workers
```

**Comment by FahaoTang** [comment]
> I was able to publish the app manually (running `dotnet publish -r linux-x64`, zipping the results and calling `az functionapp deployment source config-zip`).
> 
> I was able to cut through the breaking differences between Windows and Linux hosting (setting App Settings via Configuration) and completely silly error messages (the runtime was looking for configurations with clearly wrong keys).
> 
> According to Azure Portal, it seems like it is working, I can see all functions listed, Enabled with green checkmarks, the queue is being consumed, etc, but none of the functions work at all.
> 
> I can see the following exception in Application Insights:
> 
> ```
> Failed to start a new language worker for runtime: dotnet-isolated.
> 
> An error occurred trying to start process '/home/site/wwwroot/App.Functions' with working directory '/home/site/wwwroot'. Permission denied
> 
> System.ComponentModel.Win32Exception:
>    at System.Diagnostics.Process.ForkAndExecProcess (System.Diagnostics.Process, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
>    at System.Diagnostics.Process.StartCore (System.Diagnostics.Process, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
>    at System.Diagnostics.Process.Start (System.Diagnostics.Process, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a)
>    at Microsoft.Azure.WebJobs.Script.Workers.WorkerProcess.StartProcessAsync (Microsoft.Azure.WebJobs.Script, Version=4.24.0.0, Culture=neutral, PublicKeyToken=null: /src/azure-functions-host/src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs:64)
>    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at Microsoft.Azure.WebJobs.Script.Grpc.GrpcWorkerChannel+<StartWorkerProcessAsync>d__61.MoveNext (Microsoft.Azure.WebJobs.Script.Grpc, Version=4.24.0.0, Culture=neutral, PublicKeyToken=null: /src/azure-functions-host/src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs:281)
>    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at Microsoft.Azure.WebJobs.Script.Workers.Rpc.RpcFunctionInvocationDispatcher+<InitializeJobhostLanguageWorkerChannelAsync>d__50.MoveNext (Microsoft.Azure.WebJobs.Script, Version=4.24.0.0, Culture=neutral, PublicKeyToken=null: /src/azure-functions-host/src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs:152)
>    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at Microsoft.Azure.WebJobs.Script.Workers.Rpc.RpcFunctionInvocationDispatcher+<InitializeJobhostLanguageWorkerChannelAsync>d__49.MoveNext (Microsoft.Azure.WebJobs.Script, Version=4.24.0.0, Culture=neutral, PublicKeyToken=null: /src/azure-functions-host/src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs:144)
>    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at Microsoft.Azure.WebJobs.Script.Workers.Rpc.RpcFunctionInvocationDispatcher+<InitializeJobhostLanguageWorkerChannelAsync>d__48.MoveNext (Microsoft.Azure.WebJobs.Script, Version=4.24.0.0, Culture=neutral, PublicKeyToken=null: /src/azure-functions-host/src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs:135)
>    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
>    at Microsoft.Azure.WebJobs.Script.Workers.Rpc.RpcFunctionInvocationDispatcher+<>c__DisplayClass55_0+<<StartWorkerProcesses>b__0>d.MoveNext (Microsoft.Azure.WebJobs.Script, Version=4.24.0.0, Culture=neutral, PublicKeyToken=null: /src/azure-functions-host/src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs:215)
> ```
> 
> It consumes the queue, but all processing fails with:
> 
> ```
> Final functionDispatcher state: Initializing. Initialization timed out and host is shutting down
> Did not find any initialized language workers
> Executed 'Functions.REDACTED' (Failed, Id=080fa978-e124-4a39-9445-6524e56306e2, Duration=41051ms)
> Did not find any initialized language workers
> ```

It is the same error I got. I have converted my Azure Function app to be hosted with my website's Azure Container app environment and will update if it works properly.

**Comment by kshyju** [comment]
@FahaoTang The function [images were updated to consume .net rc1 bits](https://github.com/Azure/azure-functions-docker/pull/957), but deployment of that change has not been completed. It should be done in the next few days.

**Comment by kshyju** [comment]
Folks, 

We will use this issue to share the updates about the progress on different areas related to .NET 8 support.

If you are running into issues with .net8, **_please open a new GH issue_** with relevant details instead of leaving a comment here.  Thank you!

**Comment by aherrick** [comment]
super frustrating the bits get released but aren't publish to work in Azure makes no sense

**Comment by ShockwaverReal** [comment]
Successfully auto-installed and ran with VS2022 Version 17.8.0 Preview 2.0 | Azure Functions and Web Jobs Tools 17.8.226.21692
 against .NET8 (isolated), Windows self-contained x64

**Comment by aherrick** [comment]
things are working now as well here with RC1 on linux - is there a spot that MSFT updates to show when these updates happen?

**Comment by kshyju** [comment]
@aherrick From  .NET8 RC2 onwards, the bits will be available on azure functions within 24 hours.

**Comment by Peter-Juhasz** [comment]
@kshyju do you mean including Windows Consumption plan, or only Linux still? Thanks.

**Comment by FahaoTang** [comment]
> things are working now as well here with RC1 on linux - is there a spot that MSFT updates to show when these updates happen?

It still doesn't work for me if I use the .net 8 image to build customize azure function image as i am using Azure Container App to host instead of a dedicated host plan.

**Comment by FahaoTang** [comment]
Further update - I finally got it working while using custom image and deploy to Azure Container App.
I had to use tag `4-nightly-dotnet-isolated8.0` in MCR instead of tag `4-dotnet-isolated8.0`. Basically only the nightly tag at the moment contains .NET 8 RC 1 runtime. Can this be fixed considering rc1 is released?

I've documented details here: https://kontext.tech/article/1313/azure-function-app-failed-to-start-a-new-language-worker-for-runtime-dotnet-isolated

**Comment by kshyju** [comment]
@Peter-Juhasz Windows support is available now.
https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide#preview-net-versions

From RC2 onwards, we will provide same day support for the .NET8 bits.

**Comment by aherrick** [comment]
it's day of and Azure Functions aren't working on .NET 8 GA

**Comment by leetmike** [comment]
> it's day of and Azure Functions aren't working on .NET 8 GA - had to switch back to RC2

same. all other container images we use are updated to GA except functions. our functions app does not function ☹️

**Comment by KasprDev** [comment]
> it's day of and Azure Functions aren't working on .NET 8 GA - had to switch back to RC2

Same, I love Azure _Does-Not_ Function apps! It's almost the end of the day and it's still not been fixed or addressed.

**Comment by SeanFeldman** [comment]
You're talking to humans, not bots.
Keep it civilized.

**Comment by fabiocav** [comment]
@KasprDev , @leetmike, @aherrick to be more productive here, can you please share the hosting option (OS, model) and region you're attempting to deploy your functions to? As you can imagine, this is a massive update rolling across a large fleet, and completing this may take longer in some environments.

- Windows (all SKUs) is done
- Public images are done
- Linux (App Service plans and Elastic Premium) is done
- Linux consumption still have some regions rolling

If you can share the details, we can provide a more accurate status.

@aherrick when you provided feedback, the official .NET images were just being published, are you still seeing issues now?

**Comment by leetmike** [comment]
@fabiocav sorry for the lack of details, if Functions/.NET 8 wasn't so great we wouldn't all be so passionate! Nevertheless, the image I was using was mcr.microsoft.com/azure-functions/dotnet-isolated:4-dotnet-isolated8.0

It appears to have been updated within the last ~hour, our functions are up and running 💯 now.

**Comment by KasprDev** [comment]
> @KasprDev , @leetmike, @aherrick to be more productive here, can you please share the hosting option (OS, model) and region you're attempting to deploy your functions to? As you can imagine, this is a massive update rolling across a large fleet, and completing this may take longer in some environments.
> 
> * Windows (all SKUs) is done
> * Public images are done
> * Linux (App Service plans and Elastic Premium) is done
> * Linux consumption still have some regions rolling
> 
> If you can share the details, we can provide a more accurate status.
> 
> @aherrick when you provided feedback, the official .NET images were just being published, are you still seeing issues now?

Can confirm, functions are up and running now! Thank you!

**Comment by mattchenderson** [comment]
Thanks for confirming! Linux Consumption should now be complete for all public regions as well.

Also, just in case it catches folks off-guard, the .NET 8 templates for Functions may not show up in VS from a fresh install. But it should fetch them upon launch if connected to the internet.

**Comment by fabiocav** [comment]
Thank you all! Glad to hear things are working as expected.

**Comment by kshyju** [comment]
[Azure Functions now supports .NET 8 for applications using the isolated worker model.](https://azure.microsoft.com/en-us/updates/ga-azure-functions-supports-net-8-in-the-isolated-worker-model) 🎉

Closing this issue as this was created to track the different work items needed for shipping .net8 support. If you are experiencing issues when using .net8 dotnet isolated apps, please open a new issue with relevant details. 

Thank you!
